packages/parser/src/language/sankey/sankey.langium

```diff
@@ -20,4 +20,7 @@ SankeyLink:
 ;
 
 terminal SANKEY_LINK_VALUE returns number: /"(0|[1-9][0-9]*)(\.[0-9]+)?"|[\t ]*(0|[1-9][0-9]*)(\.[0-9]+)?/;
+/**
+ * @greedy
+ */
```

Comment:
(minor) The only way people (readers of the grammar) will know what this means is if they read and understand the code.
I'd like to see an explanatory line after this like:
```text
 *  This ensures that this rule is put at the bottom of the list of tokens.
```
And anywhere else `@greedy` is used.